langgraph/store/redis/base.py

```diff
@@ -52,6 +54,8 @@
             {"name": "key", "type": "tag"},
             {"name": "created_at", "type": "numeric"},
             {"name": "updated_at", "type": "numeric"},
+            {"name": "ttl_minutes", "type": "numeric"},
```

Comment:
I'm curious why we need to index TTL minutes and expiration times? Can we not use built in ttl handling in Redis? Or is this used to retrieve items based on some ttl range?

Reply:
TTL in the schema is for searching purposes following the PG impl